### ISSUE TITLE
Computed Navigator - Persistant on Screen on older devices

### ISSUE THREAD

**Issue by reidcanavan** [unplugged/unplugged-controls#535]
The unpComputedNavigator control will stay persistant on the phone screen after load.  Newer devices ( Android 4.4 ) do not appear to experience this behaviour.

Example code:

```
<unp:unpHeader search="none" viewsButton="true"
    title="Permits and Inspections" disablefastclick="no">
</unp:unpHeader>
<unp:unpNavigatorComputed>
    <unp:this.computedmenuitems><![CDATA[#{javascript:[
  {label: "My Schedule", hasSubMenu: true, icon: "fa-calendar", subMenu: [
    {label: "Today", ajaxloadid: "contentwrapper", ajaxtargetid: "content", hasSubMenu: false, disabled: false, page: "/UnpViewScheduleMeToday.xsp" },
    {label: "Tomorrow", ajaxloadid: "contentwrapper", ajaxtargetid: "content", hasSubMenu: false, disabled: false, page: "/UnpViewScheduleMeTomorrow.xsp" }
    ]},
  {label: "Permits", hasSubMenu: true, icon: "fa-file-text-o", subMenu: [
    {label: "By Address", ajaxloadid: "contentwrapper", ajaxtargetid: "content", page: "/UnpViewPermitsByAddress.xsp", disabled: false, hasSubMenu: false},
    {label: "By Number", ajaxloadid: "contentwrapper", ajaxtargetid: "content", page: "/UnpViewPermitsByNumber.xsp", disabled: false, hasSubMenu: false}, 
    {label: "By Type", ajaxloadid: "contentwrapper", ajaxtargetid: "content", page: "/UnpViewPermitsByType.xsp", disabled: false, hasSubMenu: false}
  ]},
  {label: "Scheduled Inspections", hasSubMenu: true, icon: "fa-calendar", subMenu: [
    {label: "Today", ajaxloadid: "contentwrapper", ajaxtargetid: "content", hasSubMenu: false, disabled: false, page: "/UnpViewScheduleToday.xsp" },
    {label: "Tomorrow", ajaxloadid: "contentwrapper", ajaxtargetid: "content", hasSubMenu: false, disabled: false, page: "/UnpViewScheduleTomorrow.xsp" }
  ]},
  {label: "Contacts", hasSubMenu: false, icon: "fa-book", ajaxloadid: "contentwrapper", ajaxtargetid: "content", disabled: false, page: "/UnpViewContacts.xsp" }
]}]]></unp:this.computedmenuitems>
</unp:unpNavigatorComputed> 

<unp:unpFlatView insetData="true" numberofrows="20" refreshmethod="pull" position="menu-aligned" viewName="v_unp_permits" xpageDoc="UnpDocPermit.xsp" title="Permits By Number" summaryColumn="permitNumber" detailColumn="cvProperty" ajaxload="Yes"></unp:unpFlatView>

<unp:unpFooter newDocLoadType="ajax" synctype="currentdb"></unp:unpFooter>
```

**Comment by whitemx** [comment]
Hi,

What devices are you testing against. I have tested on 4.4.4 (Nexus 4 / 7 / 10) and 4.3 (Galaxy SIII).

Thanks

Matt

**Comment by reidcanavan** [comment]
Galaxy Nexus and Galaxy SII LTE.

Reid

**Comment by whitemx** [comment]
Reid,

Could you let me know the version numbers for Android that these devices are running so I can set up a test environment? 

Thanks

Matt

**Comment by reidcanavan** [comment]
Galaxy nexus is running 4.3 and the SII LTE is running 4.2.2.

We have Moto X devices as well which run fine ( 4.4.3 here in Canada ).

Reid

**Comment by whitemx** [comment]
Thanks for that information. I'm now able to reproduce the problem so I will work on a fix and post it here once I understand the problem.

**Comment by reidcanavan** [comment]
Awesome.  Thanks!

Its affecting all our field users who we provide with older phones ( as
they are more prone to breaking due to the work environment ).

Reid

**Comment by tcsoft** [comment]
Hi Reid,

Can you tell me how pressing an issue this is for you, so that we can triage it into the next release.

Tim Clark
Director of Professional Services
Teamstudio

**Comment by reidcanavan** [comment]
It has stopped all our field workers from being able to use any app using
the controls ( for us it's all our apps ).  I have had to stop development
as it affects about half our users.

Reid

**Comment by whitemx** [comment]
So far tracked this down that it was working properly in v3.1 controls, but not in 3.2 or 3.3.

**Comment by whitemx** [comment]
This problem has now been resolved. It is related to a difference between the stock Android browser which no longer is used by Android (but was up to Android 4.1) and Chrome. I have added a fix to the unplugged.js Script Library, so the new version of the file can simply be copied across and the problem should be resolved.

The new version of the file can be accessed here: https://raw.githubusercontent.com/unplugged/unplugged-controls/f6c63b6c643808ac390701880e17f0fc911bafe6/nsf/Code/ScriptLibraries/unplugged.js
